diff --git a/dist/module.mjs b/dist/module.mjs
index 9f7787c0f532bc114658403ba03d14dd5c81d47d..e1292aca8dc197dcf93a2ce0076d758eb83bd58b 100644
--- a/dist/module.mjs
+++ b/dist/module.mjs
@@ -1083,7 +1083,7 @@ const ResourceProxyPlugin = createUnplugin((options = {}, meta) => {
           debug$4("load (config) ->", id, baseDir);
           const code = `import { isObject, isFunction } from '@intlify/shared'
 export default async function() {
-  const loader = await import(${toCode(withQuery(resolve$1(baseDir, query.target), { config: "true" }))}).then(m => m.default || m)
+  const loader = await import(${toCode(withQuery(resolve$1(baseDir, query.target), { hash: query.hash, config: "true" }))}).then(m => m.default || m)
   const config = isFunction(loader)
     ? await loader()
     : isObject(loader)